src/NodeManipulator/ToOneRelationPropertyTypeResolver.php

```diff
@@ -121,7 +122,7 @@ private function resolveFromDocBlock(
         DoctrineAnnotationTagValueNode $doctrineAnnotationTagValueNode
     ): Type {
         $joinDoctrineAnnotationTagValueNode = $phpDocInfo->findOneByAnnotationClass(
-            'Doctrine\ORM\Mapping\JoinColumn'
+            JoinColumn::class
```

Comment:
@TomasVotruba oops, this should be skipped as cause prefixed on downgrade.

Reply:
@TomasVotruba `StringClassNameToClassConstantRector` somehow make a change:

```diff
➜  rector-doctrine git:(fix-fixture-namespaces) time vendor/bin/rector process src/NodeManipulator/ToOneRelationPropertyTypeResolver.php --dry-run
 
 1/1 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%
1 file with changes
===================

1) src/NodeManipulator/ToOneRelationPropertyTypeResolver.php:3

    ---------- begin diff ----------
@@ @@

 namespace Rector\Doctrine\NodeManipulator;

+use Doctrine\ORM\Mapping\JoinColumn;
 use Nette\Utils\Strings;
 use PhpParser\Node\Expr;
 use PhpParser\Node\Stmt\Property;
@@ @@
         DoctrineAnnotationTagValueNode $doctrineAnnotationTagValueNode
     ): Type {
         $joinDoctrineAnnotationTagValueNode = $phpDocInfo->findOneByAnnotationClass(
-            'Doctrine\ORM\Mapping\JoinColumn'
+            JoinColumn::class
         );

         return $this->processToOneRelation(
    ----------- end diff -----------

Applied rules:
 * StringClassNameToClassConstantRector (https://wiki.php.net/rfc/class_name_scalars https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md#form)
```

so it seems somehow a bug on `fnmatch()` check 

https://github.com/rectorphp/rector-doctrine/blob/8acae8ea0de8241fd44002958646cfc3556e9209/rector.php#L33

I will check...